docs/app/pages/components/sections/date-time-picker/time-picker/time-picker.component.html

```diff
@@ -0,0 +1,122 @@
+<timepicker [ngModel]="myTime" [min]="minTime" [max]="maxTime" [hourStep]="hstep" [minuteStep]="mstep" [showMeridian]="ismeridian"
```

Comment:
This code snippet exceeds the width of the documentation page. I apologise for misleading you on the maximum line length; the correct limit is 108 characters. In VS Code you can set a rule via settings:

````
    "editor.rulers": [
        108
    ],
````